packages/firestore/test/util/node_persistence.ts

```diff
@@ -16,6 +16,7 @@
  */
 
 import * as fs from 'fs';
+// @ts-ignore
```

Comment:
Consider adding to this comment indicating what's going on. Something like, "there are no types for indexdbshim" or something like that.